README.md

```diff
@@ -5,26 +5,19 @@ Everything in this project is WIP! including this readme!
 
 # How to get up and running
 1) Install `node` and `npm` through [this link](https://docs.npmjs.com/getting-started/installing-node).
-
```

Comment:
Why did you delete this section of the README?

Reply:
These were just suggestions for ides/extensions. They're not really necessary for setting up our project :)